c_api/src/taichi_core_impl.cpp

```diff
@@ -225,7 +225,15 @@ TiRuntime ti_create_runtime(TiArch arch) {
   switch (arch) {
 #ifdef TI_WITH_VULKAN
     case TI_ARCH_VULKAN: {
-      out = (TiRuntime)(static_cast<Runtime *>(new VulkanRuntimeOwned));
+#ifdef TI_BUILD_TESTS
```

Comment:
agh we'd better separate enabling validation layer with TI_BUILD_TESTS, how about we create another macro to control validation and let TI_IN_CI control that?